frontend/src/feature/cards/components/CardDetailsComponent.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-card
+    class="pa-2 ma-2 mx-auto d-flex flex-column justify-space-between" max-width="500px" min-height="300px">
+    <v-card-title class="text-center">
+      {{ card.title }}
+    </v-card-title>
+    <v-card-subtitle>
+      <ul class="tags">
+        <li v-for="(tag, index) in card.tags">
+          <template v-if="index < card.tags!.length-1 ">
+            {{ tag }},
+          </template>
+          <template v-else>
+            {{ tag }}
+          </template>
```

Comment:
you do not need the index here, v-for just goes over all tags.
AND the use of this generic template tag makes no sense here. for a generic container use v-container or just div or span (depending if you want block or inline element)

Reply:
also: the tags are no subtitle: moven them in text part. and write label "tags" in front of them